biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/outputPanel.js

```diff
@@ -124,26 +124,7 @@ define(["jquery", "text!../settings/settings.json", "text!openPicsure/outputPane
 					this.model.set("queryRan", true);
 					this.render();
 			 	}).bind(this),
-			 	error: (function(response){
-					if (response.status === 401) {
-						history.pushState({}, "", "/psamaui/not_authorized");
-					} else {
-						this.model.set("spinning", false);
-        		                        this.model.set("queryRan", true);
-
-						// handle return of "< 10" response
-						if (response.responseText.indexOf("<") > -1) {
-							this.model.set("totalPatients", response.responseText);
-	               				        this.render();
-						} else {
-							response.responseText = "<h4>"
-							+ overrides.outputErrorMessage ? overrides.outputErrorMessage : "There is something wrong when processing your query, please try it later, if this repeats, please contact admin."
-							+ "</h4>";
-	               				        this.render();
-							$("#patient-count").html(response.responseText);
-						}
-					}
-				}).bind(this)
+				error: transportErrors.handleAll
```

Comment:
We should still stop the spinner and deal with the case that this returns and does not pushState